src/components/20-molecules/footer-small/index.scss

```diff
@@ -0,0 +1,60 @@
+.m-footer-small {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  flex-wrap: wrap;
+  padding: 15px;
+
+  background: $color-prim-blue-azure;
+  color: $color-prim-white;
+  border-top: 1px solid rgba(255, 255, 255, 0.25);
+
+  font-family: 'Source Sans Pro', Arial, sans-serif;
+  font-size: 13px;
+  font-weight: 400;
+
+  @include breakpoint($mediaquery-md-up) {
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  &__list {
+    // ### Reset browser styles for ul
+    margin: 0;
+    padding: 0;
+    text-indent: 0;
```

Comment:
is `text-indent` needed?

Reply:
dunno, i copied this from a random guy on github that claimed that this would reset an ul.